ion.js.org

trustgit opened the pull request:
- [X] There is reasonable content on the page (see: [No Content](https://github.com/js-org/js.org/wiki/No-Content))
- [X] I have read and accepted the [Terms and Conditions](http://js.org/terms.html)

indus commented:
I've already added your requested subdomain to JS.ORGs zonefile. But you have to (re-)add a file named ```CNAME``` to your branch with the single line ```ion.js.org``` to make it work.
I'll merge after we have "contact".

trustgit commented:
> I've already added your requested subdomain to JS.ORGs zonefile. But you have to (re-)add a file named `CNAME` to your branch with the single line `ion.js.org` to make it work.
> I'll merge after we have "contact".

Added the single line `CNAME` file to the repo.

indus commented:
Welcome https://ion.js.org
